projects/chaos/inc/power_path.h

```diff
@@ -0,0 +1,66 @@
+#pragma once
+// Chaos Power Path monitoring module.
+// Requires gpio, gpio interrupts, adcs, soft timers and interrupts to be initialized.
+
+#include <stdbool.h>
+#include <stdint.h>
+
+#include "adc.h"
+#include "gpio.h"
+#include "soft_timer.h"
+#include "status.h"
+
+// Conversion callbacks for ADCs.
+typedef uint16_t (*power_path_conversion_callback)(uint16_t value);
```

Comment:
style

Reply:
Done. I think there are a few other places in the codebase that need to be changed for style on callbacks to be consistent still I'll open a ticket for that. https://uwmidsun.atlassian.net/browse/ELEC-321